engine/src/main/java/org/terasology/engine/rendering/opengl/FBO.java

```diff
@@ -737,6 +740,9 @@ public void setDimensions(int width, int height) {
          * @param other A Dimension to use the width and height from.
          */
         public void setDimensions(Dimensions other) {
+            if(width == 0 && height ==0){
```

Comment:
In addition to my other review content in this case shouldn't we be checking `other.width` and same for height?